recipes/conda-auth/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "conda-auth" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda-incubator/{{ name }}/archive/refs/tags/{{ version}}.tar.gz
+  sha256: d909a24adc9856119e80ac87658ee380d007d96f022ccb27a11d6acbce72234d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

Comment:
The `[build-system]` in `pyproject.toml` says that `setuptools-scm` is required.

Reply:
Will add it.